packages/apm/src/hubextensions.ts

```diff
@@ -34,21 +34,6 @@ function startSpan(context: SpanContext | TransactionContext): Transaction | Spa
   const scope = hub.getScope();
```

Comment:
btw. why do we `// @ts-ignore` here?

Reply:
![image](https://user-images.githubusercontent.com/363802/82895843-321cb480-9f55-11ea-8243-1055e4997ff5.png)
Hub extensions hack